### DISCUSSION TITLE
`docs/src/how-to/vscode.md` links to `nix-ld-rs`, an `archived` repository

### DISCUSSION THREAD

**Discussion by hirehamir** [nix-community/NixOS-WSL#539] [General]
https://nix-community.github.io/NixOS-WSL/how-to/vscode.html links to https://github.com/nix-community/nix-ld-rs, which is an `archived` repository.

**Comment by SuperSandro2000** [comment]
Thanks #546

**Reply by hirehamir** [reply]
You're welcome!
